pkg/apis/monitoring/v1/types.go

```diff
@@ -326,6 +326,9 @@ type PrometheusSpec struct {
 	// the desired limit.
 	// Note that if SampleLimit is lower that value will be taken instead.
 	EnforcedSampleLimit *uint64 `json:"enforcedSampleLimit,omitempty"`
+	// Allow overlapping blocks in Prometheus, which in turn enables vertical compaction
```

Comment:
Nit: this should start with `AllowOverlappingBlocks ...` same as others above.

Reply:
Done.